src/elements/content-sidebar/activity-feed/annotation-thread/useReplies.js

```diff
@@ -0,0 +1,147 @@
+// @flow
+import React from 'react';
+import uniqueId from 'lodash/uniqueId';
+import APIFactory from '../../../../api';
+import commonMessages from '../../../common/messages';
+import { repliesErrors } from './errors';
+
+import type { BoxItemPermission, User } from '../../../../common/types/core';
+import type { BoxCommentPermission, Comment, FeedItemStatus } from '../../../../common/types/feed';
+import type { ElementsXhrError } from '../../../../common/types/api';
+
+type Props = {
+    annotationId: string,
+    api: APIFactory,
+    currentUser: User,
+    fileId: string,
+    filePermissions: BoxItemPermission,
+    initialReplies: Array<Comment>,
+};
+const useReplies = ({ annotationId, api, initialReplies = [], currentUser, fileId, filePermissions }: Props) => {
+    const [replies: Array<Comment>, setReplies] = React.useState(initialReplies);
```

Comment:
If needed, we can type the state using `React.useState<Array<Comment>>` rather than doing so inline here.